src/modules/core-package.ts

```diff
@@ -14,23 +14,26 @@
 
 import { homedir } from 'os';
 
-export class SdkConfig {
-    // name of the package to the package repository
-    repo: string = '';
-
+export class CorePackageConfig {
```

Comment:
Check if this class is really required, after all a package containing a core is just a regular package.